flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/IntermediateResult.java

```diff
@@ -52,6 +56,9 @@
 
     private final ResultPartitionType resultType;
 
+    private final Map<ConsumedPartitionGroup, CompressedSerializedValue<ShuffleDescriptor[]>>
```

Comment:
I prefer it to be `SerializedValue<ShuffleDescriptor[]>` and hide the implementation details of compression.

Reply:
Done.